client/signup/steps/user/index.jsx

```diff
@@ -455,6 +463,7 @@ export class UserStep extends Component {
 					recaptchaClientId={ this.state.recaptchaClientId }
 					horizontal={ isReskinned }
 					isReskinned={ isReskinned }
+					shouldDisplayUserExistsError
```

Comment:
Is there any time we _don't_ want this behavior?

Reply:
> Is there any time we _don't_ want this behavior?

Per conversation in https://github.com/Automattic/wp-calypso/pull/65908#discussion_r934743942, we always want the error to display in this context. It's a prop because we may or may not want to display the error in other contexts.